lib/database/repositories/Repository.js

```diff
@@ -53,15 +53,11 @@ class Repository {
     /**
      * Returns all entities.
      *
-     * @param {QueryBuilder} [queryBuilder] The QueryBuilder to use.
+     * @param {QueryBuilder} [findQuery] The QueryBuilder to use.
```

Comment:
When you update a parameter, take consideration of its documentation, it's almost as important as the code itself!

Reply:
done